scripts/run_benchmarks.sh

```diff
@@ -138,8 +140,8 @@ PYTHONPATH=. \
         'acceleration_framework_config_file'
 
 if [ "$DRY_RUN" = "true" ]; then 
-    echo "DRY_RUN=True, will skip compare with reference logic"
-else
+    echo "DRY_RUN=True, will skip compare with reference logic"    
+elif [ "$NO_COMPARE" = "false" ]; then
```

Comment:
lets comment this and not force a run, but instead put instructions on how to run the comparison, with specifc instructions to point to the correct reference.